### PR TITLE
✨ Implementing analytics vendor support for Memo (memo.co)

### LINKED ISSUE #25979: New analytics tool configuration for "Memo"
## Summary

Adding implementation configuration for analytics tool Memo (memo.co)

## Motivation

New integration for existing analytics product that does not have native AMP support yet.

## Additional context

Typical web integration is done via on-page JavaScript.

/cc @ampproject/wg-approvers

### PR DESCRIPTION
**Pull request by bradkagawa**
fixes #25979

This is a repeat of a previous submission, but we had problems with the CLA agreements and how they linked to git accounts. I believe this is fixed now.

/cc @zhouyx @jeffjose

### CODE REVIEW

**Comment by bradkagawa** [comment]
@avimehta I think this is everything can you confirm?

File: extensions/amp-analytics/0.1/vendors/memo.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        'maxTimerLength': 1200,
+      },
+      'request': 'timer',
+    },
+  },
+  'transport': {
+    'beacon': false,
+    'xhrpost': false,
+    'image': true,
+  },
```
Comment (micajuine-ho) [inline extensions/amp-analytics/0.1/vendors/memo.js]: To echo my last comment: just to confirm that the server only handles get request. AMP handles the fallback to xhrpost and image if the browser doesn't support one transportation method. So we'd recommend using the default transport method if that's supported by server. Thanks.

**Comment by bradkagawa** [comment]
Hi sorry yes we only want to support get. Right now posts will work but we intend to deprecate that.
________________________________

**Comment by micajuine-ho** [comment]
@bradkagawa Ok, that sounds good.

**Comment by bradkagawa** [comment]
@micajuine-ho thank you! I still need one more approval is that right?

**Pull request merged**